org.eclipse.xtext.ui.tests/src-gen/org/eclipse/xtext/ui/tests/editor/contentassist/services/ParameterizedExpressionsTestLanguageGrammarAccess.java

```diff
@@ -61,22 +61,25 @@
 		
 		//FunctionDeclaration <Yield>:
 		//	=> ({FunctionDeclaration}
-		//	-> FunctionImpl <Yield,Yield,Expression=false>) => ';'?;
+		//	-> FunctionImpl<Yield,Yield,Expression=false>) => ';'?;
```

Comment:
@ArneDeutsch Missing formatting rule for parameterized rule calls? I'd expect a space after the `,` comma